### PR TITLE
chore(test): fix npm test to install missing prerequisites

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

File: scripts/pretest.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# When running `npm test`, this script is intended to run during pretest step.
+
+WS=$(pwd)                          # the protractor directory
+$WS/bin/webdriver-manager update   # updates the local webdriver manager
+$WS/node_modules/.bin/jshint lib spec scripts
+cd $WS/website && npm install      # installs dependencies for the website
```
Comment (sjelin) [inline scripts/pretest.sh]: Why?  Are these necessary for the `npm test`?

Comment (cnishina) [inline scripts/pretest.sh]: These are required. `bin/webdriver-manager update` installs the `selenium/chromedriver` for the direct connect test. Installing the website's dev dependencies installs marked and lodash locally which are required to pass the unit test `website/docgen/spec/*.js`

**Comment by cnishina** [comment]
Refer to issue #2883

**Comment by cnishina** [comment]
Closed with 4b792280cc4c7f11b652e67123610c2b1880acf5.

**Pull request closed without merging**